assets/styles/components.scss

```diff
@@ -1042,8 +1043,12 @@ h1 {
     width: 15rem;
   }
 
-  input + *, .input-group + * {
-    margin-block-start: var(--spacing-card-md);
+  >,
+  .extend-styling> {
+    input + *,
+    .input-group + * {
+      margin-block-start: var(--spacing-card-md);
+    }
   }
```

Comment:
Kind of unrelated to this PR, but I really dislike this styling block. It's not super readable and there's probably better ways to do this